apps/charterafrica/src/pages/index.page.js

```diff
@@ -42,6 +42,23 @@ export async function getStaticProps({ defaultLocale, locale, locales }) {
     fallbackLocale: defaultLocale,
   });
 
+  const { docs: homePage } = await payload.findPage("home", {
+    locale,
+    fallbackLocale: defaultLocale,
+  });
+
+  const { blocks: pageBlocks } = homePage[0] ?? {
+    blocks: [],
+  };
+
+  const blocks =
+    pageBlocks?.map(({ blockType, ...other }) => ({
+      ...other,
+      slug: blockType,
+    })) || [];
+
+  const ecosystem = blocks.find((block) => block.slug === "ecosystem");
+
```

Comment:
@kilemensi Awesome. I think 
`const ecosystem = blocks.find((block) => block.slug === "ecosystem") || null` 
should instead be 
`const ecosystem = blocks.find((block) => block.slug === "ecosystem") || {};` 
because the first one will throw an error when [looping through slugs](https://github.com/CodeForAfrica/ui/blob/c3693944400a373683a8b8df063b566e1367cda2/apps/charterafrica/src/pages/index.page.js#L14): `Cannot read properties of null (reading 'slug')`

Reply:
I see your point @kelvinkipruto. I prefer `null` to `{}` for empty values so I'd have changed https://github.com/CodeForAfrica/ui/blob/abc7ce406fe3a9cc7ba48b8882dfec4d61002ef5/apps/charterafrica/src/pages/index.page.js#L14 to `block?slug`.

Your call though.

Reply:
Yes. This still works. 🚀